tests_system/lobster_trlc/test_conversion_errors.py

```diff
@@ -1,5 +1,3 @@
-from typing import Optional
-from dataclasses import dataclass
 from .lobster_system_test_case_base import LobsterTrlcSystemTestCaseBase
 from ..asserter import Asserter
```

Comment:
Now that we have renamed hyphens to underscores
these imports can also be changed to

```suggestion
from tests_system.asserter import Asserter
from tests_system.lobster_trlc.lobster_system_test_case_base import LobsterTrlcSystemTestCaseBase
```